packages/next/client/image.tsx

```diff
@@ -697,54 +756,132 @@ export default function Image({
   useEffect(() => {
     handleLoading(imgRef, srcString, layout, placeholder, onLoadingCompleteRef)
   }, [srcString, layout, placeholder, isVisible])
-
+  const imgElementArgs = {
+    lazy: isLazy,
+    ...{
```

Comment:
Maybe I'm not understanding this, but couldn't you just assign these props instead of spreading with `...`?

Reply:
Yeah I think there was a conditional that got lost in the shuffle at some point. fixed